src/medley/core.cljc

```diff
@@ -70,6 +70,16 @@
       (persistent! (reduce-kv (f assoc!) (transient (empty coll')) coll'))
       (reduce-kv (f assoc) (empty coll') coll'))))
 
+(defn- completing-rf
+  "Turns function of 3 arguments [rf result x] into transducing function
+  returning 0, 1, and 2-arity functions."
```

Comment:
The function is private, so the docstring should be omitted.

Reply:
Okay, I didn't know of this convention - wouldn't a docstring still be helpful for someone reading the code to understand what it does? Should I put it in a comment instead?

Reply:
It's in the contributing guidelines. A docstring generally exists for public APIs, which isn't useful for private functions. A comment should explain "why", whereas the code should be clear enough to make the "what" obvious.

Reply:
Thanks for the explanation! I'll remove it then.